tests/test_smoketest.py

```diff
@@ -111,6 +111,22 @@ def test_smoketest(self):
             run(f"{venv_bin}hca dss get-subscriptions --replica {replica}")
             run(f"{venv_bin}hca dss delete-subscription --replica {replica} --uuid {sub_id}")
 
+        run(f"{venv_bin}hca dss post-bundles-checkout "
+            "--uuid 00251ba0-39fc-4afe-9249-891a3de0f5ba "
```

Comment:
Why is there a hardcoded UUID here? I think this should be the uuid of the bundle that was uploaded by smoketest earlier (`--uuid $(jq -r .bundle_uuid upload.json)`).

Reply:
fixed